### ISSUE TITLE
Support for devices without hcitool

### ISSUE THREAD

**Issue by AngellusMortis** [mKeRix/room-assistant#187]
**Is your feature request related to a problem? Please describe.**

`hcitool` is now deprecated and will likely start getting removed in future versions of Raspbian and other OSes. It is already removed from the official Manjaro ARM RPI4 image. 

https://git.kernel.org/pub/scm/bluetooth/bluez.git/commit/?id=b1eb2c4cd057624312e0412f6c4be000f7fc3617

**Describe the solution you'd like**

A solution that does not require the deprecated tools.

**Describe alternatives you've considered**

For now, it is possible to still get `hcitool` and the other deprecated tools with the `--enable-deprecated` flag at compile time. For Manjaro/Archlinux ARM this means just installing the `bluez-utils-compat` AUR package, but that may not be a viable solution forever.

**Comment by mKeRix** [comment]
I actually stumbled upon this issue while I was working on the Alpine-based Docker image and then tried out the current CLI tool `bluetoothctl` - unfortunately the implementation for getting the RSSI there didn't allow the use case that room-assistant needs though from what I remember. I agree that we should move away from deprecated packages though. If anyone has an idea on how to achieve the same cc & rssi combination (without having to pair devices!) please do let me know.

**Comment by paulrbeers** [comment]
Yes please do not require me to pair my device to all my nodes!    That was the reason I left Monitor.    And whatever the solution is, it has to support both BLE AND Classic as I don't want to have to teach my wife how to start up an iBeacan app every time she restarts her iphone.   She barely tollerates my home automation as it is! (that's why I stopped trying to use esp32-mqtt-home).

I used both of those before I came here!

**Comment by jun3280net** [comment]
@mKeRix - I'm unable to get your app to work for my setup (Linux Mint VM on Win10 Host, with Home Assistant Supervisor running in the Linux Environment - integrated bluetooth).  I'm not very conversant in Linux, and trying to figure out how to get this to work with my setup (I tried MONITOR as well and kept getting ("cycling bluetooth hardware on hci0").  

That said, I stumbled across a thread regarding **bluetootctl and RSSI** - which may help : 
[https://stackoverflow.com/questions/36607626/how-to-use-bluetoothctl-like-hcitool-lescan-to-report-repeated-proximity-beacons](https://stackoverflow.com/questions/36607626/how-to-use-bluetoothctl-like-hcitool-lescan-to-report-repeated-proximity-beacons)

Cheers thanks for your application
